docs/src/modules/branding/BrandingSupportTheProject.tsx

```diff
@@ -0,0 +1,151 @@
+import * as React from 'react';
+import Container from '@material-ui/core/Container';
+import { experimentalStyled as styled } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardHeader from '@material-ui/core/CardHeader';
+import CardContent from '@material-ui/core/CardContent';
+import Avatar from '@material-ui/core/Avatar';
+import Typography from '@material-ui/core/Typography';
+import Grid from '@material-ui/core/Grid';
+import Link from '@material-ui/core/Link';
+import Button from '@material-ui/core/Button';
+import QuestionAnswerIcon from '@material-ui/icons/QuestionAnswer';
+import UnderlinedText from 'docs/src/modules/branding/UnderlinedText';
+import FeedbackIcon from 'docs/src/modules/branding/icons/Feedback';
+import ChangesIcon from 'docs/src/modules/branding/icons/Changes';
+import FinanceIcon from 'docs/src/modules/branding/icons/Finance';
+import HelpIcon from 'docs/src/modules/branding/icons/Help';
+import OpenCollectiveIcon from 'docs/src/modules/branding/icons/OpenCollective';
+
+interface SupportCardProps {
+  color?: 'primary' | 'info';
+  icon?: React.ReactNode;
+  children?: React.ReactNode;
+  title?: string;
+}
+
+const SupportCard = styled((props: SupportCardProps) => {
+  const { color, icon, title, children, ...other } = props;
+  return (
+    <Card elevation={0} {...other}>
+      <CardHeader avatar={<Avatar aria-label={title}>{icon}</Avatar>} />
+      <CardContent>
+        <Typography variant="h3">{title}</Typography>
+        <Typography component="div">{children}</Typography>
+      </CardContent>
+    </Card>
```

Comment:
What do you think about removing the usage of the Card? It seems to create more issues than to solve ones.

Reply:
For instance, the left alignment that should exist:

<img width="418" alt="gap" src="https://user-images.githubusercontent.com/3165635/104358460-c51f5480-550e-11eb-822b-15070262c1d1.png">

Or the spacing between items that is 32px instead of a smaller expected value

Reply:
Yeah, will remove it 👍 Also the spacings were anyway not as expected by default..

Reply:
Removed and separated the component to `BrandingCard`